Makefile

```diff
@@ -1,8 +1,8 @@
 REPO ?= github.com/osbkit/minibroker
 BINARY ?= minibroker
 PKG ?= $(REPO)/cmd/$(BINARY)
-REGISTRY ?= osbkit/
-IMAGE ?= $(REGISTRY)minibroker
+REGISTRY ?= splatform/
```

Comment:
Will using this suse-specific value make acceptance upstream more difficult ?

Reply:
Most likely, yes. At the same time, we have this already https://github.com/SUSE/minibroker/blob/suse-cf/charts/minibroker/values.yaml#L26.

Reply:
You should probably base your branch on the _upstream_ branch, and then modify it for us in a PR to merge the upstream changes?